src/Security/MemberAuthenticator/LostPasswordForm.php

```diff
@@ -23,8 +25,15 @@ class LostPasswordForm extends MemberLoginForm
      */
     public function getFormFields()
     {
+        $uniqueIdentifier = Member::config()->get('unique_identifier_field');
+        if ($uniqueIdentifier === 'Email') {
+            $emailField = EmailField::create('Email', _t('SilverStripe\\Security\\Member.EMAIL', 'Email'));
+        } else {
+            // This field needs to still be called Email, but we can re-label it
+            $emailField = TextField::create('Email', _t('SilverStripe\\Security\\Member.LOSTPASSWORDLABEL', $uniqueIdentifier));
+        }
         return FieldList::create(
-            EmailField::create('Email', _t('SilverStripe\\Security\\Member.EMAIL', 'Email'))
+            $emailField
         );
```

Comment:
May as well consolidate this into a single line now
```suggestion
        return FieldList::create($emailField);
```